client/components/AttributionEditor/AttributionEditor.tsx

```diff
@@ -58,7 +59,9 @@ class AttributionEditor extends Component<Props> {
 
 	handleAttributionAdd(user) {
 		const { attributions, onUpdateAttributions, onPersistStateChange } = this.props;
-		const maxOrder = attributions.length ? Math.max(...attributions.map((a) => a.order)) : 0;
+		const maxOrder = attributions.length
+			? Math.max(...attributions.map((a) => a.order ?? 0))
```

Comment:
`a.order` can be `null`, think setting it to `0` would be best, but should be smoke tested